src/jsMain/kotlin/app/softwork/bootstrapcompose/Classes.kt

```diff
@@ -0,0 +1,90 @@
+package app.softwork.bootstrapcompose
+
+public object BSClasses {
+    public const val active: String = "active"
+
+    public val align: Align = Align
+    public val background: Background = Background
+    public val display: Display = Display
+
+    public const val collapse: String = "collapse"
+    public const val navbar: String = "navbar"
+    public const val navbarBrand: String = "navbar-brand"
+    public const val navbarCollapse: String = "navbar-collapse"
+    public const val navLink: String = "nav-link"
+    public const val navbarNav: String = "navbar-nav"
+    public const val navbarToggler: String = "navbar-toggler"
+    public const val navbarTogglerIcon: String = "navbar-toggler-icon"
+}
+
+public object Align {
+    public const val baseline: String = "align-baseline"
+    public const val top: String = "align-top"
+    public const val middle: String = "align-middle"
+    public const val bottom: String = "align-bottom"
+    public const val textTop: String = "align-text-top"
+    public const val textBottom: String = "align-text-bottom"
+}
+
+public object Background {
```

Comment:
What is the use case of this class? Alternative: add a `background()` function to `Color` to prevent 2 color classes

Reply:
The idea is to be able to specify background color classes without having to use constant strings in your code. I added background() and text() functions to Color so that class could be used to generate strings for either set of classes.